app/assets/javascripts/host_checkbox.js

```diff
@@ -97,19 +97,22 @@ function cleanHostsSelection() {
 
 function multiple_selection() {
   var total = $("#pagination").data("count");
-  var alert_text = Jed.sprintf(__("All <b> %d </b> hosts are selected. "), total);
+  var alert_text = Jed.sprintf(n__("Single host is selected in total",
+                                   "All <b> %d </b> hosts are selected.", total), total);
   var undo_text = __("Undo selection");
   var multiple_alert = $("#multiple-alert");
-  multiple_alert.find(".text").html(alert_text + '<a href="#" onclick="undo_multiple_selection();">' + undo_text + '</a>');
+  multiple_alert.find(".text").html(alert_text + '<a href="#" onclick="undo_multiple_selection();"> ' + undo_text + '</a>');
```

Comment:
Still need to move the space here (and below) to outside the `<a>` tag:
![screenshot from 2017-04-23 12-07-57](https://cloud.githubusercontent.com/assets/1540531/25312265/f13c218c-281d-11e7-8ac2-56d9e4e13059.png)